src/components/Reports/ExpectedMonthlyTotalReport/Table/ExpectedMonthlyTotalReportTable.tsx

```diff
@@ -17,70 +17,61 @@ import {
 import { ExpandMore } from '@material-ui/icons';
 import { useTranslation } from 'react-i18next';
 import theme from '../../../../theme';
-
-export interface Contact {
-  name: string;
-  contactId: string;
-  status: string;
-  commitment: string;
-  frequency: string;
-  converted: string;
-  currency: string;
-  donation?: string;
-}
+import { ExpectedDonationRowFragment } from 'pages/accountLists/[accountListId]/reports/GetExpectedMonthlyTotals.generated';
 
 interface Props {
+  accountListId: string;
   title: string;
-  data: Contact[];
+  data: ExpectedDonationRowFragment[];
   donations: boolean;
+  total: number;
+  currency: string;
 }
 
 export const ExpectedMonthlyTotalReportTable: React.FC<Props> = ({
+  accountListId,
   title,
   data,
   donations,
+  total,
+  currency,
 }) => {
   const { t } = useTranslation();
 
-  const totalPartners = () => {
-    return 6;
-  };
-
   const [visible, setVisible] = useState(true);
 
-  const total = () => {
-    return 3920;
-  };
-
   const showTotalPartners = () => {
     if (visible) {
-      return t(' Show ' + totalPartners() + ' Partners');
+      return t(
+        ' Show ' + data.length + ` Partner${data.length > 1 ? 's' : ''}`,
+      );
     }
   };
-  return (
+
+  const isEmpty = data.length === 0;
```

Comment:
Actually you can use this variable to help with the logic for `showTotalPartners`.